x/tokenfactory/simulation/sim_msgs.go

```diff
@@ -13,43 +13,98 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-var (
-	TokenFactoryCreationFee = sdk.NewCoins(sdk.NewInt64Coin("stake", 10_000_000))
-)
-
 // RandomMsgCreateDenom creates a random tokenfactory denom that is no greater than 44 alphanumeric characters
 func RandomMsgCreateDenom(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk.Context) (*types.MsgCreateDenom, error) {
-	minCoins := TokenFactoryCreationFee
+	minCoins := sim.TokenFactoryKeeper().GetParams(ctx).DenomCreationFee
 	acc, err := sim.RandomSimAccountWithMinCoins(ctx, minCoins)
 	if err != nil {
-		return nil, err
+		return nil, errors.New("no account has enough coins to pay the denom creation fee")
```

Comment:
suggest also wrapping the actual `err`

Reply:
lol I have no idea why I did this for some of these, thank you!